charts/finops-agent/templates/deployment.yaml

```diff
@@ -147,13 +146,12 @@ spec:
             {{- if .Values.agent.cloudability.enabled }}
             - name: CLOUDABILITY_CLUSTER_NAME
               value: {{ (include "finops-agent.clusterId" .) | quote }}
-            - name: CLOUDABILITY_EMITTER_ENABLED
+            - name: CLOUDY_EMITTER_ENABLED
```

Comment:
we could add a check for both in code as well for compatibility reasons if you're worried about this!

Reply:
why not. what's one more var?!

Reply:
🧇